user_agents/parsers.py

```diff
@@ -165,7 +165,7 @@ def is_tablet(self):
             return True
         if self.os.family.startswith('Windows RT'):
             return True
-        if self.os.family == 'Firefox OS' and 'Mobile' not in self.browser.family:
+        if self.os.family == 'Firefox OS' and 'Tablet;' in self.ua_string:
```

Comment:
With the latest uap-core changes all Android Firefox-es, tablet and mobile, get browser.family="Firefox Mobile" so either way _is_android_tablet has to change, correct?